src/applications/disability-benefits/bdd/BDDApp.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import RoutedSavableApp from '../../../platform/forms/save-in-progress/RoutedSavableApp';
```

Comment:
There is a eslint rule being added that will show the leading `../` (in front of "platform") as an error. So please remove all of these now.

Reply:
Oops, yeah good point. I'll fix that. Too much copy/pasting without paying attention.